services/headless-lms/models/src/exercise_task_submissions.rs

```diff
@@ -210,6 +219,34 @@ WHERE ets.id = $1
     CourseOrExamId::from(res.course_id, res.exam_id)
 }
 
+pub async fn get_peer_reviews_received(
+    conn: &mut PgConnection,
+    exercise_id: Uuid,
+    user_id: Uuid,
+) -> ModelResult<PeerReviewsRecieved> {
+    let peer_review =
+        crate::peer_review_configs::get_by_exercise_id(&mut *conn, exercise_id).await?;
```

Comment:
I think we should use here `get_by_exercise_or_course_id` so that we'll handle the case too where the peer review config used is a course default peer review config.